preamble.tex

```diff
@@ -57,7 +47,7 @@
 \newcommand{\tat}{{\raise.17ex\hbox{$\scriptstyle\mathtt{\sim}$}}} %tilde
 \newcommand{\tac}{\textasciicircum} %XOR
 \newcommand{\te}[1]{\texttt{#1}} %for programming font
-\newcommand{\lra}{\Longrightarrow} % ==>
+\newcommand{\lra}{\implies} % ==>
```

Comment:
I think it would just make sense to grep and replace all usages of `\lra` with `\implies`, since `\lra` would just convolute this macro

Reply:
agreed